demo/index.html

```diff
@@ -16,11 +7,18 @@
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
     <link rel="import" href="../ll-token-field.html">
   </head>
-  <body>
+  <body style="margin: 30px;">
 
     <p>An example of <code>&lt;ll-token-field&gt;</code>:</p>
 
-    <ll-token-field tags='["1", "2", "3", "4"]'></ll-token-field>
+    <div style="height: 100px; border:1px solid gainsboro; border-radius: 4px; padding: 5px;">
```

Comment:
Yes, I wanted the hosting item to control the border. For display purposes, I added  a wrapper with these styles to the demo.

I might be wrong with this implementation, but it did work for our first use case. If you have any ideas on this, I am open to them.